modules/statics/src/coins.ts

```diff
@@ -15922,6 +15930,15 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset['sol:moveusd'],
     SOL_TOKEN_FEATURES
   ),
+  solToken(
+    'a5beabb1-bd50-43e3-8f69-a4622c43ac63',
+    'sol:dxl',
+    'Dexlab',
+    8,
```

Comment:
Update the Decimal to 6.
https://solscan.io/token/GsNzxJfFn6zQdJGeYsupJWzUAm57Ba7335mfhWvFiE9Z

Reply:
Updated it. Thanks!